app/src/main/java/de/westnordost/streetcomplete/data/meta/OsmTaggings.kt

```diff
@@ -32,32 +32,38 @@ val MAXSPEED_TYPE_KEYS = setOf(
 
 const val SURVEY_MARK_KEY = "check_date"
 
-val KEYS_THAT_SHOULD_NOT_BE_REMOVED_WHEN_SHOP_IS_REPLACED = listOf(
-    "landuse", "historic", "historic:.*", "heritage", "heritage:.*",
-    // building/simple 3d building mapping
-    "building", "man_made", "building:.*", "roof:.*", "architect",
-    // any address (or weird address: is_in??)
-    "addr:.*", "is_in:.*",
-    // shop can at the same time be an outline in indoor mapping
-    "level", "level:ref", "indoor", "indoor:.*", "room",
-    // geometry
-    "layer", "ele", "height", "area", "is_in",
-    // notes and fixmes
-    "FIXME", "fixme", "note",
-    // reference tags that are specific to the location and are unlikely to change when a business changes
-    // due to the multitude of these, let's keep the whole ref:.* space
-    "ref:.*",
-    "source",
-    // source tags referring to building / address / geometry (from cursory look at taginfo on "source:")
-    "source:(${listOf(
-        "date",
-        // referring to address / ref...
-        "addr", "address", "housenumber", "street", "conscriptionnumber", "streetnumber", "postcode", "postal_code", "ref",
-        // referring to geometry
-        "geometry", "position", "shape", "outline", "location",
-        // referring to building
-        "building", "heritage", "height", "architect", "ele"
-    ).joinToString("|")})(:.*)?",
+// generated by "make update" from https://github.com/mnalis/StreetComplete-taginfo-categorize/
+val KEYS_THAT_SHOULD_BE_REMOVED_WHEN_SHOP_IS_REPLACED = listOf(
+    "shop_?[1-9]?(:.*)?", "craft_?[1-9]?", "amenity_?[1-9]?", "old_amenity", "old_shop", "information", "leisure", "office", "tourism",
+    // "popular" shop=* / craft=* subkeys
+    "marketplace", "household", "swimming_pool", "telecommunication", "laundry", "golf", "sports", "ice_cream", "scooter", "music", "retail", "yes", "ticket", "health", "newsagent", "lighting", "truck", "car_repair", "video", "fuel", "farm", "car", "ski", "sculptor", "hearing_aids", "surf", "photo", "boat", "gas", "kitchen", "anime", "builder", "hairdresser", "security", "bakery", "fishing", "doors", "kiosk", "market", "bathroom", "lamps", "vacant", "insurance", "caravan", "gift", "bicycle", "insulation", "communication", "mall", "model", "empty", "wood", "hunting", "motorcycle", "trailer", "camera", "water", "fireplace", "outdoor",
```

Comment:
could you use about ~max 100 characters per line for better readability?

Reply:
Added auto-wrap at 100 chars.